backend/build.gradle.kts

```diff
@@ -276,6 +278,19 @@ tasks.named<Checkstyle>("checkstyleTest") {
     configFile = rootProject.file("checkstyle/config-test.xml")
 }
 
+tasks.named<DependencyUpdatesTask>("dependencyUpdates").configure {
+    rejectVersionIf {
+        candidate.group == "io.sentry" || (isNonStable(candidate.version) && !isNonStable(currentVersion))
+    }
```

Comment:
is this overwriting the config from line 157 (`rejectVersionIf { !isStable(candidate.version) }`? if yes I think we should remove one of the two configurations

Reply:
this is probably the reason that unstable versions where added again

Reply:
Oh true I completely oversaw that config. Moving the sentry ignore to that part